src/overlays/actors/ovl_En_Ani/z_en_ani.h

```diff
@@ -6,14 +6,41 @@
 struct EnAni;
 
 typedef void (*EnAniActionFunc)(struct EnAni*, GlobalContext*);
+typedef void (*EnAniBlinkFunc)(struct EnAni*);
+
+#define ANI_LIMB_COUNT 16
 
 typedef struct EnAni {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x1B0];
-    /* 0x02F4 */ EnAniActionFunc actionFunc;
-    /* 0x02F8 */ char unk_2F8[0x8];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ ColliderCylinder collider1;
+    /* 0x190 */ ColliderCylinder collider2;
+    /* 0x1DC */ SkelAnime skelAnime;
+    /* 0x220 */ Vec3s jointTable[ANI_LIMB_COUNT];
+    /* 0x280 */ Vec3s morphTable[ANI_LIMB_COUNT];
+    /* 0x2E0 */ Vec3s headRot;
+    /* 0x2E6 */ Vec3s unk2E6;
+    /* 0x2EC */ u16 stateFlags;
+    /* 0x2EE */ s16 unusedShort;
+    /* 0x2F0 */ s16 treeReachTimer;
+    /* 0x2F4 */ EnAniBlinkFunc blinkFunc;
+    /* 0x2F8 */ s16 blinkState;
+    /* 0x2FA */ s16 blinkTimer;
+    /* 0x2FC */ EnAniActionFunc actionFunc;
 } EnAni; // size = 0x300
 
-extern const ActorInit En_Ani_InitVars;
+enum EnAniType {
+  /* 0 */ ANI_TYPE_UNUSED_STANDING, // other, not really zero
```

Comment:
Again, for a general-purpose actor like this I don't really like `unused`.

Reply:
Can I at least leave a comment saying its unused? I don't want people to come to this actor and think this state is working and normally found in the game.